modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/PartitionCommand.java

```diff
@@ -18,31 +18,19 @@
 package org.apache.ignite.internal.table.distributed.command;
 
 import java.util.UUID;
+
+import org.apache.ignite.internal.table.distributed.TableMessageGroup;
+import org.apache.ignite.network.NetworkMessage;
+import org.apache.ignite.network.annotations.Transferable;
 import org.apache.ignite.raft.client.WriteCommand;
-import org.jetbrains.annotations.NotNull;
 
 /**
  * Partition transactional command.
  */
-public abstract class PartitionCommand implements WriteCommand {
-    /** Transaction id. */
-    private UUID txId;
-
-    /**
-     * The constructor.
-     *
-     * @param txId Transaction id.
-     */
-    public PartitionCommand(@NotNull UUID txId) {
-        this.txId = txId;
-    }
-
+@Transferable(TableMessageGroup.Commands.PARTITION)
```

Comment:
Can this be transferred by its own? The class used to be abstract, maybe we don't need a dedicated message type, no one would use it anyway

Reply:
You're right, I removed PARTITION message type from message group and Transferrable annotation here.